pkg/apis/certmanager/v1alpha2/types_certificate.go

```diff
@@ -145,6 +150,21 @@ type CertificateSpec struct {
 	KeyEncoding KeyEncoding `json:"keyEncoding,omitempty"`
 }
 
+type X509DistinguishedName struct {
+	// Country/Region
+	Country            []string `json:"country,omitempty"`
+	Organization       []string `json:"organization,omitempty"`
+	OrganizationalUnit []string `json:"organizationalUnit,omitempty"`
+	// City
+	Locality []string `json:"locality,omitempty"`
+	// State/Province
+	Province      []string `json:"province,omitempty"`
+	StreetAddress []string `json:"streetAddress,omitempty"`
+	PostalCode    []string `json:"postalCode,omitempty"`
```

Comment:
Can you pluralise all of these `[]string` types? It's been a common complaint that our `spec.organization` field is actually a slice despite being singular 🙈